app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] You should have a validation for the user's name attribute. Name must not be blank

Reply:
I have added validation to user model class. Thanks for the input.